rasa/cli/test.py

```diff
@@ -115,29 +116,51 @@ def run_core_test(args: argparse.Namespace) -> None:
     )
 
 
-def run_nlu_test(args: argparse.Namespace) -> None:
-    """Run NLU tests."""
+async def run_nlu_test_async(
+    config: Optional[Union[Text, Dict]],
+    data_path: Optional[Text],
+    models_path: Optional[Text],
+    output_dir: Optional[Text],
+    cross_validation: Optional[bool],
+    percentages: Optional[List[int]],
+    runs: Optional[int],
+    no_errors: Optional[bool],
+    all_args: Optional[Dict[Text, Any]],
+) -> None:
+    """Runs NLU tests.
+
+    Args:
+        all_args: all arguments gathered in a Dict so we can pass them to 'perform_nlu_cross_validation' and
```

Comment:
can please wrap the docstring so that it doesn't exceed our 88 char limit?

Reply:
I did! I also changed the setting in Pycharm so that the wrapping line is at 88 characters and this hopefully does not happen again.